src/ClientApp/src/services/ContentService.ts

```diff
@@ -0,0 +1,55 @@
+import { ContentKey } from './ContentKey';
+import { AllContent, Language } from './Models';
+
+export class ContentService {
+
+    private content: Array<{ [key: string]: string; }> = [];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Private member variable 'content' is never reassigned; mark it as 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS2933)